src/posts/2022-11-11-summit-highlights/index.md

```diff
@@ -0,0 +1,14 @@
+---
+date: "2022-11-11T17:00:00.000Z"
+title: "Highlights from the eBPF* Summit 2022"
```

Comment:
Minor nit: `eBPF*` -> `eBPF`

Reply:
I left it that way because it is actually how they have it in the blog title. Would you prefer it fixed?

<img width="1065" alt="Screenshot 2022-11-14 at 10 41 44" src="https://user-images.githubusercontent.com/29048870/201688462-a713426b-c35b-47ef-a422-0769b4b09cf8.png">

Reply:
(yeah, imho, we should, looks weird otherwise)